src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -74,7 +74,7 @@
           query: playlistIdFinal ? {playlistId: playlistIdFinal} : {}
         }"
       >
-        {{ displayTitle }}
+        {{ title }}
```

Comment:
Using `displayTitle` is fine, `this.title` is used
```suggestion
        {{ displayTitle }}
```